sources/yioapi.cpp

```diff
@@ -166,6 +166,10 @@ bool YioAPI::addEntity(QVariantMap entity) {
         if (obj) {
             IntegrationInterface *integration = qobject_cast<IntegrationInterface *>(obj);
 
+            if (!integration) {
+                qCWarning(CLASS_LC) << "integration pointer is null" << entity.value(Config::KEY_ENTITY_ID);
+            }
```

Comment:
See above. If an entity must have an integration, then it shouldn't be added to m_entities.